web-ui/view/organisation.ml

```diff
@@ -1,24 +1,33 @@
 open Git_forge
 
 module Make (M : M_Git_forge) = struct
-  let profile_picture_url org =
-    (* FIXME [benmandrew]: How can we get the GitLab profile pictures? *)
-    match M.prefix with
-    | "github" -> Printf.sprintf "https://github.com/%s.png?size=200" org
-    | "gitlab" -> "/images/gitlab-logo-500.png"
-    | _ -> ""
+  let profile_picture org =
+    (* /profile-pictures is where images are downloaded -- see Dockerfile.web *)
+    let local_image =
+      Printf.sprintf "/profile-pictures/%s/%s.png" M.prefix org
+    in
+    let fallback_image = Printf.sprintf "/images/%s-logo-500.png" M.prefix in
+    let local_image_exists =
+      try
+        let _ = Unix.stat local_image in
+        true
+      with _ -> false
```

Comment:
```suggestion
    let local_image_exists = Bos.OS.File.exists (Fpath.v local_image) in
```
As we use `Bos` in `ocurrent`, this is a simple suggestion to avoid dealing with the exception. Feel free to ignore it.